### DISCUSSION TITLE
m1 맥북에서 안드로이드 스튜디오 설치시

### DISCUSSION THREAD

**Discussion by lemonit96** [velopert/dealing-with-react-native#10] [Q&A] [answered]
안녕하세요? m1 맥북에서 학습을 진행중인데 기존에 깔아놓은 android studio로 yarn android 실행시 문제가 생겨서 (제 생각에는 에뮬 문제인것 같습니다.) 아예 삭제하고 다시 다운받으려고 합니다. 이때 실리콘버전으로 설치를 해야되는지 intel 버전으로 설치해야 되는지 잘 모르겠어서 질문 남깁니다. 
그리고 추가적으로 아직 m1 제대로된 에뮬이 지원 안하는걸로 알고있는데 교재에서는 어떻게 해결하신건지도 궁금합니다.

**Comment by velopert** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/17202261/147684765-970596b9-f484-44f4-ab8d-1a3062a8b790.png)
![image](https://user-images.githubusercontent.com/17202261/147684777-a018929d-27f6-47aa-a32f-d5d3f64e2329.png)


arm64_ 가 붙어있는 이미지들을 사용하면 됩니다!  예전엔 지원이 안됐었는데, 현재 정식으로 지원을 하고있어요.
